python/lsst/ts/cbp/mock_server.py

```diff
@@ -0,0 +1,119 @@
+__all__ = ["MockServer"]
+
+import asyncio
+import re
+import logging
+
+
+class MockServer:
+    def __init__(self, log=None):
+        self.host = "127.0.0.1"
+        self.port = 9999
+        self.timeout = 5
+        self.long_timeout = 30
+        self.azimuth = 0
+        self.altitude = 0
+        self.focus = 0
+        self.mask = 1
+        self.panic_status = False
+        self.encoders = [False, False, False, False, False, False]
+        self.park = False
+        self.auto_park = False
+        self.masks = {1: 0, 2: 0, 3: 0, 4: 0, 5: 0}
+        self.command_calls = {
+            "az": self.format_azimuth,
```

Comment:
Please consider naming these do_x instead of format_x. Right now all they do is issue a reply, but in the long run they might do something more sophisticated. For example it would be nearly trivial to add simulated point-to-point actuators from ts_simactuators and command them to move. This gives:
* realistic motion delays
* knowledge of the current position (e.g. for telemetry)
* support for aborting motion
* limits handling